include/client.h

```diff
@@ -1269,6 +1269,19 @@ class Client : public SRObject
                                                     const int start_index,
                                                     const int end_index);
 
+        /*!
+        *   \brief Reconfigure the chunking size that Redis uses for model
+        *          serialization, replication, and the model_get command.
+        *   \details This method triggers the AI.CONFIG method in the Redis
+        *             database to change the model chunking size. The default
+        *             size of 511MB should be fine for most applications, so
+        *             it is expected to be very rare that a client calls this
+        *             method.
+        *   \param chunk_size The new chunk size
```

Comment:
I'm pretty sure the unit is bytes, but let's call that out specifically in the docstring

Reply:
Addressed in all 6 places where needed